src/App.js

```diff
@@ -47,11 +111,11 @@ function App() {
                     <ErrorPage resetSearch={resetSearch} />
                 </Route>
             </Switch>
-        </AppStyled>
+        </Wrapper>
     )
 }
 
-const AppStyled = styled.div`
+const Wrapper = styled.div`
```

Comment:
Maybe ``AppWrapper`` or just ``App`` would work as well and additionally describe what they wrap or are.  Or ``AppContainer`` ?